examples/cms-umbraco-heartcore/tailwind.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = {
+  purge: ['./components/**/*.js', './pages/**/*.js'],
```

Comment:
Can we add JIT mode please?

Reply:
Updated config to match the other examples